com.unity.template.renderstreaming-rtx/Packages/manifest.json

```diff
@@ -6,11 +6,13 @@
     "com.unity.ide.rider": "1.1.4",
     "com.unity.ide.vscode": "1.1.4",
     "com.unity.inputsystem": "1.0.0-preview.4",
-    "com.unity.render-pipelines.high-definition": "7.1.8",
+    "com.unity.render-pipelines.high-definition": "7.3.1",
+    "com.unity.render-pipelines.high-definition-config": "file:../LocalPackages/com.unity.render-pipelines.high-definition-config",
```

Comment:
Why this package is in local?

Reply:
It's setup by HDRP package automatically.
They said that, it is required to let users set up compile-time settings, instead of copying the whole HDRP package locally.

Reply:
I am not sure why HDRP team decided this design but there's nothing we can do about it...
https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@8.0/manual/HDRP-Config-Package.html